nailgun/entities.py

```diff
@@ -267,41 +267,39 @@ def path(self, which=None):
             )
         return super(ActivationKey, self).path(which)
 
-    def add_subscriptions(self, payload):
+    def add_subscriptions(self, synchronous=True, **kwargs):
```

Comment:
Why do we add `synchronous=True` to all helper functions? As far as I remember we need that parameter to manage asynchronous tasks.
Do we have so serious changes in the application logic that all of these operations become asynchronous?